scripts/ci/release_version_cal.py

```diff
@@ -139,6 +139,11 @@ def clean_mod_of_azdev(mod):
     result = subprocess.run(cmd, stdout=subprocess.PIPE)
     if result.returncode:
         raise Exception(f'Error when removing {mod} from azdev')
+    # remove extension source code
+    cmd = ["rm", "-rf", "./src/" + mod]
```

Comment:
If the mod name and dir name are inconsistent, will an exception be thrown here?

Reply:
No, this step just clears the module loading from source code, for afterwards loading module from `whl`.

We can check the mod package name and folder name in a different CI task.